src/gps_spoofer.py

```diff
@@ -58,21 +70,41 @@ def callback_odometry(self, data):
     #print "pos_var = ", self.pos_var
 
     self.pwc.header.stamp = rospy.Time.now()
-    self.pwc.pose.pose = data.pose.pose
-    
-    # TODO: Take height measurement from pressure sensor
+
+    # Pose message
+#    self.pwc.pose.pose = data.pose.pose
+    x = data.pose.pose.position.x
+    y = data.pose.pose.position.y
+
+    #angle = radians(-7.0) #imu compass
+    angle = -2.73
+
+    # Rotate x and y to align x to East
+    self.pwc.pose.pose.position.x = cos(angle)*x - sin(angle)*y
+    self.pwc.pose.pose.position.y = sin(angle)*x + cos(angle)*y
+
+    # TODO: Take height measurement from laser
     self.pwc.pose.pose.position.z = data.pose.pose.position.z
+    self.pwc.pose.pose.orientation = self.latest_imu_message.orientation
+
+    # Point message
+    # Take x and y from vicon, take altitude from laser
+    self.point.header = self.pwc.header
+    self.point.point.x = x
+    self.point.point.y = y
+    self.point.point.z = self.altitude.data
```

Comment:
Okay, as this is a single param I'll add it to the launch file directly. Let me know if you think we need a .yaml.